shell/src/main/java/alluxio/cli/fsadmin/FileSystemAdminShellUtils.java

```diff
@@ -37,6 +46,35 @@ public static int compareTierNames(String a, String b) {
     return aValue - bValue;
   }
 
+  /**
+   * @return whether master client service is running
+   */
+  public static boolean masterClientServiceIsRunning() {
+    // Check if Alluxio master and client services are running
+    try (CloseableResource<FileSystemMasterClient> client =
+        FileSystemContext.INSTANCE.acquireMasterClientResource()) {
+      MasterInquireClient inquireClient = null;
+      try {
+        InetSocketAddress address = client.get().getAddress();
+        List<InetSocketAddress> addresses = Arrays.asList(address);
+        inquireClient = new PollingMasterInquireClient(addresses, () ->
+            new ExponentialBackoffRetry(50, 100, 2));
+      } catch (UnavailableException e) {
+        System.err.println("Failed to get the leader master.");
```

Comment:
it's not good for a utils method to have side effects like this. Usually utils methods don't have side effects, and for something like printing to System.err/System.out especially, it's better to do all the printing from a central location so that its easy to reason about what will get printed to the user. Could we change this to something like
```java
public static void checkMasterClientServiceRunning() {
  if (!isRunning()) {
    throw new IOException("Failed to get the leader master. Please check your Alluxio master status");
  }
}
```

Reply:
Changed, thanks